ras/diag_encl.py

```diff
@@ -43,6 +45,7 @@ def run_cmd_out(cmd):
                                      ignore_status=True,
                                      sudo=True).decode("utf-8")
 
+    @skipIf(not IS_POWER_NV, "Skipped on KVM system")
```

Comment:
may be this string needs to re worded as "Skipped Test case test supported in PowerNV system"